src/Promenade.Data/PromenadeContext.cs

```diff
@@ -32,13 +32,13 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         public DbSet<Navigation> Navigations { get; }
         public DbSet<NavigationText> NavigationTexts { get; }
         public DbSet<Page> Pages { get; }
-        public DbSet<PageHeader> PageHeaders { get; }
         public DbSet<SiteSetting> SiteSettings { get; }
         public DbSet<SocialCard> SocialCards { get; }
         public DbSet<UrlRedirect> UrlRedirects { get; }
 
         // Read/Write 
         public DbSet<Language> Languages { get; set; }
         public DbSet<UrlRedirectAccess> UrlRedirectAccesses { get; set; }
+        public DbSet<PageHeader> PageHeaders { get; set; }
```

Comment:
Why does PageHeaders need to be writable here? @iafb @madison-boman

Reply:
Those writes are used for setting the values in the context from the database. We'll need to come up with a different way of restricting writes to the database.

Reply:
I see.